v2/components/schemas/InterfaceArray.yaml

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: array
+description: Modernized interface response
+items:
+  type: object
+  properties:
```

Comment:
Check `interfaceJSON`, `plugJSON` and `slotJSON`. There are missing fields for `slots` and `plugs`.

Reply:
The schemas should already be there through `Plug.yaml` and `Slot.yaml`